README.md

```diff
@@ -101,3 +44,23 @@ Make sure to do the following:
 * Update version numbers in the respective SDKs
 * Make a new release on Github with name according to version number
 
+### Extensions
+
+**Work-In-Progress**
+
+Extensions to `keelson` can be of three different types:
+* additional well-known subjects/payloads/messages, i.e. a `message` extension
+* additional generic RPC interfaces, i.e. a `interface` extension
+* additional connector implementations, i.e. a `connector` extension
+
+For convenience, extensions should:
+* be named as: `keelson-<extension_type>-<extension_name>`, for example `keelson-connector-mavlink`
+* add a topic `keelson-<extension_type>` to its repository to be visible
```

Comment:
What for topic?

Reply:
https://github.com/topics

Will add this link to the README to further clarify.

Reply:
This one is better: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics